lib/cocoapods/installer/xcode/pods_project_generator/pod_target_installer.rb

```diff
@@ -311,6 +318,9 @@ def create_test_xcconfig_files
                   test_target_bc.base_configuration_reference = xcconfig_file_ref
                 end
               end
+
+              # also apply the private config to resource bundle test targets.
+              apply_xcconfig_file_ref_to_resource_bundle_targets(target.test_resource_bundle_targets, xcconfig_file_ref)
```

Comment:
now this is properly setting the test xcconfig for resource bundle targets used by tests.